thesis/future.tex

```diff
@@ -0,0 +1,38 @@
+\chapter{Future}
+This chapter will describe the short and long term plans for future development of the product.
+
+\section{Short term}
```

Comment:
I think it is ok to make this chapter more prose instead of the sections, as it's mostly considerations thoughts.
It becomes a lot of head lines and not so much text like this.